tests/common/plugins/loganalyzer/__init__.py

```diff
@@ -15,10 +16,15 @@ def loganalyzer(duthost, request):
         logging.info("Log analyzer is disabled")
         yield
         return
+
     # Force rotate logs
-    duthost.shell(
-        "/usr/sbin/logrotate -f /etc/logrotate.conf > /dev/null 2>&1"
-        )
+    try:
+        duthost.shell(
+            "/usr/sbin/logrotate -f /etc/logrotate.conf > /dev/null 2>&1"
+            )
+    except RunAnsibleModuleFail as e:
+        logging.warning("logrotate: duplicate log entry in configuration files")
```

Comment:
I think the reason of failure can be different. I'd just log exception message.

Reply:
Done.